imap_processing/idex/idex_l1a.py

```diff
@@ -131,9 +131,9 @@ def __init__(self, packet_file: Union[str, Path], data_version: str) -> None:
 
         dust_events = {}
         for packet in decom_packet_list:
-            if "IDX__SCI0TYPE" in packet.data:
-                scitype = packet.data["IDX__SCI0TYPE"].raw_value
-                event_number = packet.data["IDX__SCI0EVTNUM"].derived_value
+            if "IDX__SCI0TYPE" in packet.user_data:
+                scitype = packet.user_data["IDX__SCI0TYPE"].raw_value
+                event_number = packet.user_data["IDX__SCI0EVTNUM"]
```

Comment:
```suggestion
                event_number = packet.user_data["IDX__SCI0EVTNUM"].raw_value
```
?

Reply:
The packet items are the derived value now. If there was no "derived" / "calibration" going on, then it is identical to the raw_value. So this should be left without the `raw_value` attribute.

Reply:
This is my understanding as well. That's why I removed `.derived_value` where it was used before.